hail/src/main/scala/is/hail/utils/ShowStrConfig.scala

```diff
@@ -0,0 +1,6 @@
+package is.hail.utils
+
+
+case class ShowStrConfig(
```

Comment:
Do we want to expose these to the user in the Python version of `.show`?

Reply:
yes, but that's a separate change and I plan to do it in a separate PR

Reply:
also want some feedback on what is configurable!